File: exercicios professor/revisao2matematica.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VetorVetor
{
    class Program
    {
        //o software deve possuir um vetor inicial, com valores já alocados
        int[] vetor = {1,2,3,4,5,6,7,8,9,10};

        public void mostraVetor(int[] vet) {
            for(int index = 0; index < vet.Length; index++)
            {
                Console.Write("\t" + vet[index]);
            }
            Console.WriteLine("");
        }

        static void Main(string[] args)
        {
            
            //instancio a classe
            Program programa = new Program();
            //vetor para armazenar os valores digitados pelo usuário
            int[] vetorUsuario = new int[programa.vetor.Length];

            for (int x=0; x < programa.vetor.Length; x++)
            {
                Console.WriteLine("Digite um valor inteiro: ");
                vetorUsuario[x] = Int32.Parse(Console.ReadLine());                
            }

            Console.WriteLine("Digite a operação: (soma, sub, mult, div):");
            String op = Console.ReadLine();
            
            for(int y=0; y < programa.vetor.Length; y++)
            {
                if (op.Equals("soma")) { programa.vetor[y] = programa.vetor[y] + vetorUsuario[y]; }
                if (op.Equals("sub")) { programa.vetor[y] = programa.vetor[y] - vetorUsuario[y]; }
                if (op.Equals("mult")) { programa.vetor[y] = programa.vetor[y] * vetorUsuario[y]; }
                if (op.Equals("div")) { programa.vetor[y] = programa.vetor[y] / vetorUsuario[y]; }

            }

            programa.mostraVetor(programa.vetor);
            //para ficar esperando o fim da execução
            Console.ReadLine();
        }
    }
}

File: exercicios professor/revisao2matematica.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package vetorvetor;

import java.util.Scanner;

/**
 *
 * @author usrlab09
 */
public class Principal {

    int[] vetor = {1,2,3,4,5,6};
    
    public void mostraVetor(int[] vet){
        for(int index=0;index<vet.length;index++){
            System.out.print("\t"+vet[index]);
        }
        System.out.println("");
    }
    
    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
            //Instancia objeto Principal
            Principal programa = new Principal();
            Scanner teclado = new Scanner(System.in);
            int[] vetorUsuario = new int[programa.vetor.length];
            
            for(int x=0;x<programa.vetor.length;x++){
                System.out.println("Digite um valor inteiro:");
                vetorUsuario[x] = teclado.nextInt();
            }
            System.out.println("Digite a operação: (soma,sub,mult,div):");
            String op = teclado.nextLine();
            
            for(int y=0;y<programa.vetor.length;y++){
                if(op.equals("soma")){programa.vetor[y]= programa.vetor[y]+vetorUsuario[y];}
                if(op.equals("sub")){programa.vetor[y]= programa.vetor[y]-vetorUsuario[y];}
                if(op.equals("mult")){programa.vetor[y]= programa.vetor[y]*vetorUsuario[y];}
                if(op.equals("div")){programa.vetor[y]= programa.vetor[y]/vetorUsuario[y];}
            }
            
            programa.mostraVetor(programa.vetor);
    }
    
}